app/bars/BarsConnector.scala

```diff
@@ -66,11 +66,13 @@ class BarsConnector @Inject() (
               if (barsError.code == BarsError.sortCodeOnDenyList) {
                 BarsResponseSortCodeOnDenyList(barsError)
               } else {
-                val msg = new RuntimeException(s"Unhandled error code for 400 HttpResponse: [$barsError]")
-                logger.error("couldn't validateBank", msg)
-                throw new RuntimeException(msg)
+                val ex = new RuntimeException(s"Unhandled error code for 400 HttpResponse: [$barsError]")
```

Comment:
if this exception were just thrown here then you'd automatically get a stack trace that you can see in kibana - we probbaly don't need additional logging here cos the exception message and the stack trace should be enough info

Reply:
removed manual logging